### PR TITLE
@ashfurrow => [Mobile] Add note about using the right Eigen signing identity.

### PR DESCRIPTION
**Pull request by alloy**
Does this satisfy your wishes?

### CODE REVIEW

**Comment by dblock** [comment]
@ashfurrow bump

**Comment by ashfurrow** [comment]
Thanks dB!

File: mobile/mobile.md [hunk 1]
```diff
@@ -31,7 +31,9 @@ Current members of the Mobile Team can be found on the [Artsy/Mobile](https://gi
...
 * GitHub notifications, etc. in [#mobile-notifications](https://artsy.slack.com/messages/mobile-notifications/).
-* Certificates and keys for signing apps for development/distribution are in the Engineering 1Password vault. The certs/keys are stored as a developer identity that can be imported into Xcode; the entry is named "Apple iOS Mobile Developer Identity". Make sure to install from there instead of letting Xcode revoke the existing keys. 
+* Certificates and keys for signing apps for development/distribution are in the Engineering 1Password vault. The certs/keys are stored as a developer identity that can be imported into Xcode; the entry is named "Apple iOS Mobile Developer Identity". Make sure to install from there instead of letting Xcode revoke the existing keys.
+
+  For more information related to choosing the right certificate and profile for development builds (that include Push Notification support), see [this document](https://github.com/artsy/eigen/blob/master/docs/push_notifications.md). In case you need to manually install the cright certificate, it is called “Eigen Developer Identity” in the Engineering 1Password vault.
```
Comment (alloy) [inline mobile/mobile.md]: I had to crigh a little :cry:

Comment (ashfurrow) [inline mobile/mobile.md]: Haha, my bad.

**Pull request merged**